tests/quic.c

```diff
@@ -1142,6 +1142,46 @@ static int test_quic_server_hello(int verbose) {
 
 #ifdef HAVE_SESSION_TICKET
 
+static int test_quic_key_share(int verbose) {
+    WOLFSSL_CTX *ctx_c, *ctx_s;
+    int ret = 0;
+    QuicTestContext tclient, tserver;
+    QuicConversation conv;
+
+    AssertNotNull(ctx_c = wolfSSL_CTX_new(wolfTLSv1_3_client_method()));
```

Comment:
Where are `ctx_c` and `ctx_s` free'd?

Reply:
Added them in the last push. Thanks for noticing.